### PR TITLE
[Merged by Bors] - Update join testnet instructions

### PR DESCRIPTION
**Pull request by avive**
## Motivation
The instructions on how to join any testnet and TweedleDee in the readme were wrong. Now they are correct.

## Changes
Just readme instructions change.

## Test Plan
Tested joining TweedeleDee with the updated instructions.

## TODO

- [x] Explain motivation or link existing issue(s)
- [x] Test changes and document test plan
- [x] Update documentation as needed

### CODE REVIEW

**Comment by avive** [comment]
bors try

File: README.md [hunk 1]
```diff
@@ -139,7 +139,7 @@ You specify these parameters by providing go-spacemesh with a toml config file.
 Assuming tn1.toml is a testnet config file saved in the same directory as go-spacemesh, use the following command to join the testnet. The data folder will be created in the same directory as go-spacemesh. The node will use TCP port 7152 and UDP port 7152 for p2p connections:
 
 ```bash
-./go-spacemesh --grpc-server --json-server --tcp-port 7152 --config tn1.toml -d sm_data
+./go-spacemesh --grpc-server --json-server --tcp-port 7152 --config ./tn1.toml -d ./sm_data
```
Comment (y0sher) [inline README.md]:
```suggestion
./go-spacemesh --grpc-server --json-server --tcp-port 7152 --config ./[tn1.toml](http://ae7809a90692211ea8d4d0ea80dce922-597797094.us-east-1.elb.amazonaws.com/) -d ./sm_data
```

**Review by y0sher — APPROVED** [review]
added small suggestion

**Comment by avive** [comment]
bors try

**Comment by avive** [comment]
bors merge

**Comment by avive** [comment]
bors merge

**Pull request closed without merging**